### PR TITLE
Adding some notes about editor SLAs

### PR DESCRIPTION
**Pull request by arfon**
What do you think @openjournals/joss-editors ?

### CODE REVIEW

File: docs/submitting.md [hunk 1]
```diff
@@ -332,17 +332,21 @@ After submission:
 
 - An Associate Editor-in-Chief will carry out an initial check of your submission, and proceed to assign a handling editor.
 - The handling editor will assign two or more JOSS reviewers, and the review will be carried out in the [JOSS reviews repository](https://github.com/openjournals/joss-reviews).
-- Authors will respond to reviewer-raised issues (if any are raised) on the submission repository's issue tracker. Reviewer and editor contributions, like any other contributions, should be acknowledged in the repository.
+- Authors will respond to reviewer-raised issues (if any are raised) on the submission repository's issue tracker. Reviewer and editor contributions, like any other contributions, should be acknowledged in the repository. 
+- **JOSS reviews are iterative and conversational in nature.** Reviewers are encouraged to post comments/questions/suggestions in the review thread as they arise, and authors expected to respond in a timely fashion.
+- Authors and reviewers are asked to be patient when waiting for a response from an editor. Please allow two weeks for an editor to respond to a question before prompting them for further action.
```
Comment (danielskatz) [inline docs/submitting.md:337]: I feel like 2 weeks is really slow, unless an editor is out of the office.  I could imagine this leading our average review time to increase a lot, if editors decide that they can generally wait this long.

File: docs/submitting.md [hunk 2]
```diff
@@ -332,17 +332,21 @@ After submission:
...
+### A note about 
+
 ## Confidential requests
 
 Please write admin@theoj.org with confidential matters such as retraction requests, report of misconduct, and retroactive author name changes.
 
 In case of a name change, the DOI will be unchanged and the paper will be updated without publishing a correction notice or notifying co-authors.
 
-JOSS will also update CrossRef metadata.
+JOSS will also update CrosrRef metadata.
```
Comment (danielskatz) [inline docs/submitting.md:352]:
```suggestion
JOSS will also update Crossref metadata.
```

File: docs/submitting.md (hunk 1 above)
Comment (arfon) [inline docs/submitting.md:337]:
```suggestion
- Authors and reviewers are asked to be patient when waiting for a response from an editor. Please allow a week for an editor to respond to a question before prompting them for further action.
```

**Comment by kyleniemeyer** [comment]
Sure, looks reasonable to me

**Review by adonath — APPROVED** [review]
Thanks @arfon, looks good to me!

**Comment by majensen** [comment]
👍Sent from my iPhoneOn Apr 24, 2023, at 8:05 AM, Kyle Niemeyer ***@***.***> wrote:﻿
Sure, looks reasonable to me

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are on a team that was mentioned.Message ID: ***@***.***<
 /span>.com>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/openjournals/joss/pull/1144#issuecomment-1520029793",
"url": "https://github.com/openjournals/joss/pull/1144#issuecomment-1520029793",
"name": "View Pull Request"
},
"description": "View this Pull Request on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]-- 
This message has been scanned for viruses and
dangerous content by
MailScanner, and is
believed to be clean.


-- 
This message has been scanned for viruses and
dangerous content by
MailScanner, and is
believed to be clean.

**Comment by oliviaguest** [comment]
looks sensible to me! 😊

File: docs/submitting.md [hunk 3]
```diff
@@ -332,7 +332,9 @@ After submission:
 
 - An Associate Editor-in-Chief will carry out an initial check of your submission, and proceed to assign a handling editor.
 - The handling editor will assign two or more JOSS reviewers, and the review will be carried out in the [JOSS reviews repository](https://github.com/openjournals/joss-reviews).
-- Authors will respond to reviewer-raised issues (if any are raised) on the submission repository's issue tracker. Reviewer and editor contributions, like any other contributions, should be acknowledged in the repository.
+- Authors will respond to reviewer-raised issues (if any are raised) on the submission repository's issue tracker. Reviewer and editor contributions, like any other contributions, should be acknowledged in the repository. 
+- **JOSS reviews are iterative and conversational in nature.** Reviewers are encouraged to post comments/questions/suggestions in the review thread as they arise, and authors expected to respond in a timely fashion.
```
Comment (arfon) [inline docs/submitting.md:336]:
```suggestion
- **JOSS reviews are iterative and conversational in nature.** Reviewers are encouraged to post comments/questions/suggestions in the review thread as they arise, and authors are expected to respond in a timely fashion.
```

**Pull request merged**